services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/BusinessRule.java

```diff
@@ -167,5 +167,4 @@ public void setDesc(String desc) {
     Invalidation,
     BoosterNotification
   }
-
```

Comment:
I guess this is a unwanted change in this class

Reply:
there was an extra space. when i added a new RuleType i made changes, but then i saw it's not necessary. if you want the extra space, i can roll it back :)